cmd/options.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
add some unit test for this? Also, since we are not running unit test for `./cmd` package yet due to main.go was the only file earlier, as we are adding more file, could you make changes to run unit test for `./cmd` as part of the `make test` target.